.github/workflows/patch-release-build.yml

```diff
@@ -0,0 +1,100 @@
+# Releases a patch by cherrypicking commits into a release branch based on the previous
+# release tag.
+name: Patch Release Build
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.1, 1.2.2
+        required: true
+      commits:
+        description: Comma separated list of commit shas to cherrypick
```

Comment:
In case we need a patch that can't be done with a simple cherrypick, do we have a description of what that process would look like? An example would be a bugfix change that was done against the main branch, but where the main branch APIs have changed from what was on the release branch.

Reply:
Great callout - I added docs for this case.